src/DB/Migration/Migration20211228T1640692399.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\DB\Migration;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\DB\Table\ShippingRateTable;
+use wpdb;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class Migration20211228T1640692399
+ *
+ * @package Automattic\WooCommerce\GoogleListingsAndAds\DB\Migration
+ *
+ * @since x.x.x
+ */
+class Migration20211228T1640692399 extends AbstractMigration {
+
+	/**
+	 * @var ShippingRateTable
+	 */
+	protected $shipping_rate_table;
+
+	/**
+	 * Migration constructor.
+	 *
+	 * @param wpdb              $wpdb The wpdb object.
+	 * @param ShippingRateTable $shipping_rate_table
+	 */
+	public function __construct( wpdb $wpdb, ShippingRateTable $shipping_rate_table ) {
+		parent::__construct( $wpdb );
+		$this->shipping_rate_table = $shipping_rate_table;
+	}
+
+
+	/**
+	 * Returns the version to apply this migration for.
+	 *
+	 * @return string A version number. For example: 1.4.1
+	 */
+	public function get_applicable_version(): string {
+		return 'x.x.x';
```

Comment:
I assume this is also temporary and will be manually changed, since the version replace in WR won't catch this.

Reply:
Hmm. I think we can add a step in the release document to modify this value because there's no way to know this before release.

Reply:
Yes that sounds like a good idea.

Reply:
Added a new step to the preparing for release section of our wiki: https://github.com/woocommerce/google-listings-and-ads/wiki/Release-Process